lib/js/tests/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap__test.js

```diff
@@ -10,7 +11,7 @@ Webapi__Dom__DomStringMap.set("fooKey", "barValue", dataset);
 
 Webapi__Dom__DomStringMap.get("fooKey", dataset);
 
-Webapi__Dom__DomStringMap.unsafeDeleteKey("fooKey", dataset);
+Curry._2(Webapi__Dom__DomStringMap.unsafeDeleteKey, "fooKey", dataset);
```

Comment:
Note that if you compile with the latest BuckleScript, this does not have runtime currying.